frontend/src/metabase/nav/components/StoreLink/StoreLink.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { t } from "ttag";
+import Tooltip from "metabase/components/Tooltip";
+import { StoreIcon, StoreIconRoot, StoreIconWrapper } from "./StoreLink.styled";
+
+const StoreLink = () => {
+  return (
+    <Tooltip tooltip={t`Explore paid features`}>
+      <StoreIconRoot href="https://metabase.com">
```

Comment:
The exact link TBD.

Reply:
Let's go with `https://metabase.com/upgrade`

Reply:
And you don't need to wait for that URL to be up to merge this in, it will be live by the time we ship 41.

Reply:
Got it now, thanks! I'll make the changes shortly